react/lib/components/Widget/WidgetContainer.tsx

```diff
@@ -174,19 +185,27 @@ export const WidgetContainer: React.FunctionComponent<WidgetContainerProps> =
               txSound.play().catch(() => {});
             }
 
-            const currencyTicker = getCurrencyTypeFromAddress(to);
             if (!hideToasts)
               enqueueSnackbar(
                 `${
                   successText ? successText + ' | ' : ''
                 }Received ${receivedAmount} ${currencyTicker}`,
-                snackbarOptions,
+                snackbarOptionsSuccess,
               );
 
             setSuccess(true);
             onSuccess?.(transaction);
           } else {
             onTransaction?.(transaction);
+            if (transactionText){
+              enqueueSnackbar(
+                `${
+                  transactionText ? transactionText : 'New transaction'
```

Comment:
I don't think this should be set by default.

Let's keep the old behaviour (of no toast) if it's not set.